src/tokenize.js

```diff
@@ -548,13 +548,38 @@ pp.readString = function(quote) {
 
 // Reads template string tokens.
 
+const INVALID_TEMPLATE_ESCAPE_ERROR = {}
+
+pp.tryReadTemplateToken = function() {
```

Comment:
It looks like [finishToken](https://github.com/ternjs/acorn/blob/b1cc4ee21af5d36718d651435eb2a2ca94445df4/src/tokenize.js#L179-L187) doesn't return anything though, so I think `readTmplToken` would return `undefined` anyway.

(I'm not trying to argue that there should be no return statement -- I just want to make sure I'm understanding what effects it would have, if any.)